common/beerocks/bwl/dwpal/ap_wlan_hal_dwpal.cpp

```diff
@@ -2442,6 +2438,8 @@ bool ap_wlan_hal_dwpal::process_dwpal_event(char *buffer, int bufLen, const std:
 
         msg->params.channel   = son::wireless_utils::freq_to_channel(msg->params.frequency);
         msg->params.bandwidth = dwpal_bw_to_beerocks_bw(chan_width);
+        LOG(DEBUG) << "channel = " << int(msg->params.channel)
+                   << " bw = " << int(msg->params.bandwidth);
```

Comment:
We have this info printed on STA_Connected event .
please remove this print.

Reply:
Removed